resources/build/bootstrap-ubuntu-20.04.sh

```diff
@@ -22,8 +22,8 @@ sudo apt-get install -y --no-install-recommends libfontenc-dev libice-dev libsm-
 
 # Install additional build tools.
 sudo pip3 install -r "$WORKSPACE/resources/build/requirements.txt"
-# Use explicit predictable conan root path, to be used for both packages
-# and conan CMake toolchain config.
+# Use explicit predictable conan root path, where packages are 
+# installed.
```

Comment:
[consider] `where packages are cached` - as we go on to use `--install-folder` in a few lines, and its not to this dir.  Conan documents [this var](https://docs.conan.io/en/latest/reference/env_vars.html#conan-user-home) as being for the cache dir.

Reply:
[6fd9e26](https://github.com/OpenAssetIO/OpenAssetIO/pull/665/commits/6fd9e26fd530240f8704c2a1146dd282253e215b) && [2213989](https://github.com/OpenAssetIO/OpenAssetIO/pull/665/commits/2213989fb7dd9a59e779fbc284df9a4243d29e88)